package.json

```diff
@@ -20,7 +20,7 @@
     "tailwind"
   ],
   "volta": {
```

Comment:
node `18.17.0` is now available. It comes with npm `9.6.7`. Then we are aligned with current node LTS (for the next two weeks or so 😅

Reply:
Updated to 18.17 & 9.6.7 respectively.